presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixTableProperties.java

```diff
@@ -232,6 +238,17 @@ public PhoenixTableProperties()
         return Optional.of(value);
     }
 
+    public static Optional<String> getDataBlockEncoding(Map<String, Object> tableProperties)
+    {
+        requireNonNull(tableProperties);
+
+        String value = (String) tableProperties.get(DATA_BLOCK_ENCODING);
+        if (value == null) {
+            return Optional.empty();
+        }
+        return Optional.of(value);
```

Comment:
Can we use `Optional.ofNullable`

Reply:
I fixed all the occurrences here, as that just seemed right and no functional change at all.